sormas-ui/src/main/java/de/symeda/sormas/ui/campaign/campaigndata/CampaignDataView.java

```diff
@@ -120,24 +212,27 @@ public CampaignFormDataFilterForm createFilterBar() {
 						.findFirst()
 						.orElse(null);
 				}
-
-				List<CampaignFormElement> formListElements =
-					formMeta.getCampaignFormElements().stream().filter(CampaignFormElement::isImportant).collect(Collectors.toList());
-				for (CampaignFormElement element : formListElements) {
-					String caption = null;
-					if (translations != null) {
-						caption = translations.getTranslations()
-							.stream()
-							.filter(t -> t.getElementId().equals(element.getId()))
-							.map(CampaignFormTranslation::getCaption)
-							.findFirst()
-							.orElse(null);
+				final boolean onlyImportantFormElements = CampaignFormElementImportance.IMPORTANT.equals(campaignFormElementImportance.getValue());
+				final List<CampaignFormElement> campaignFormElements = formMeta.getCampaignFormElements();
+				for (CampaignFormElement element : campaignFormElements) {
+					if ((onlyImportantFormElements && element.isImportant()) || !onlyImportantFormElements) {
```

Comment:
`if (element.isImportant() || !onlyImportantFormElements)` is easier and equivalent.